x/blob/types/blob_tx.go

```diff
@@ -37,46 +31,46 @@ func NewBlob(ns namespace.ID, blob []byte) (*Blob, error) {
 	}, nil
 }
 
-// ProcessBlobTx performs stateless checks on the BlobTx to ensure that the
+// ValidateBlobTx performs stateless checks on the BlobTx to ensure that the
 // blobs attached to the transaction are valid. During this process, it
 // separates the blobs from the MsgPayForBlob, which are returned in the
 // ProcessedBlobTx.
```

Comment:
This function no longer returns a `ProcessedBlobTx`.

```suggestion
// ValidateBlobTx performs stateless checks on the BlobTx to ensure that the
// blobs attached to the transaction are valid.
```

Reply:
Thanks